src/container/__tests__/RelayContainer-test.js

```diff
@@ -25,12 +25,9 @@ describe('RelayContainer', function() {
   var React;
   var ReactTestUtils;
   var Relay;
-  var RelayMetaRoute;
```

Comment:
Again, we sincerely appreciate your effort in making these fixes, but we can't accept the PR unless there are zero changes to files in `src/`.